higlass/client.py

```diff
@@ -432,6 +446,12 @@ def from_dict(cls, conf):
             uid=conf.get("uid", None),
             overlays=conf.get("overlays", []),
         )
+
+        if "genomePositionSearchBox" in conf:
+            self.conf["genomePositionSearchBox"] = json.loads(
+                json.dumps(conf["genomePositionSearchBox"])
```

Comment:
This looks like an antipattern. What is this roundtripping through JSON meant to achieve? Deep copying?

Reply:
Yes, deep copying, but since it's meant to be serializable as a string, it's actually a functional anti-pattern :-P